resources/views/components/forum/post.blade.php

```diff
@@ -11,6 +11,26 @@ class="post__datetime"
         >
             {{ $post->created_at?->diffForHumans() }}
         </time>
+
+        @if ($post->updated_at > $post->created_at)
+            <span class="post__lastupdate">
+                (last edited
+                <time datetime="{{ $post->updated_at }}" title="{{ $post->updated_at }}">
+                    {{ $post->updated_at?->diffForHumans() }}
+                </time>
+                by
+                <a
+                    class="user-tag__link {{ $post->updatedBy->group->icon }}"
+                    href="{{ route('users.show', ['user' => $post->updatedBy]) }}"
+                    style="color: {{ $post->updatedBy->group->color }}"
+                    title="{{ $post->updatedBy->group->name }}"
+                >
+                    {{ $post->updatedBy->username }}
+                </a>
```

Comment:
Let's use the `<x-user_tag>` component here instead.

Reply:
The user tag does not work, since it requires the ìs_donor` or `is_lifetime` which we do not have in this view and IMO don't want to bloat it with that too (due to mobile view)?

Reply:
Hmm, that might actually be a problem. Is there enough room for the user to be placed in this location? Make sure to try all the different resolutions and make sure it works. it might need to be moved somewhere else. Perhaps take inspiration of github's new/old "edited" comment ui.

Don't forget to eager load `updatedBy.group` on all views that load posts, otherwise those queries are ran 100s of times per page.

Reply:
It will need to be eager loaded inside the HomeController, /user/PostController, /Livewire/PostSearch, and /Livewire/TopicPostSearch

Reply:
Updated view and added eager loading